dali/pipeline/data/tensor_list_test.cc

```diff
@@ -488,8 +465,10 @@ TYPED_TEST(TensorListTest, TestShareData) {
 
   // Share the data
   tensor_list2.ShareData(&tensor_list);
-  tensor_list2.Resize({{kernels::TensorShape<>{tensor_list.size()}}});
-  tensor_list2.template mutable_data<uint8>();
+  // We need to use the same size as the underlying buffer
+  // N.B. using other type is UB in most cases
+  tensor_list2.Resize({{{tensor_list.size()}}});
```

Comment:
Not sure, but shouldn't it be
```suggestion
  tensor_list2.Resize({{{tensor_list.num_elements()}}});
```
?

Reply:
The size() is a Buffer call, that returns number of elements as well (but not the size in bytes).